chips/README.md

```diff
@@ -0,0 +1,294 @@
+# Chip components
+
+Chips appear dynamically as a group of multiple interactive elements. With chips, users can filter
+content, enter information, make selections or trigger actions.
+
+## Chip design specs
+
+You can find the design specs on [decathlon.design](https://www.decathlon.design/).
+
+## Usage
+
+If you want to use components of this module in your android mobile application, you should
+first add the Gradle dependency in your Gradle file:
+
+```kotlin
+implementation("com.decathlon.vitamin.compose:chips:<versions>")
+```
+
+### Action
+
+Action chip offers actions related to the content.
+It appears dynamically and contextually in the interface.
+
+```kotlin
+object VitaminChips {
+    
+    @Composable
+    fun Action(
+        label: String,
+        modifier: Modifier = Modifier,
+        enabled: Boolean = true,
+        colors: VitaminChipColors = VitaminChipDefaults.default(),
```

Comment:
The name of the object `VitaminChipDefaults` is now `VitaminChipColors`.

Reply:
REadme updated